testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/security/RunAsPrincipalTestCase.java

```diff
@@ -218,4 +232,40 @@ private Throwable checkEjbException(Throwable ex) {
         }
         return ex;
     }
+
+    private boolean logContainsWARN() {
+        String jbossHome = System.getProperty("jboss.inst");
+        String logPath = String.format("%s%sstandalone%slog%sserver.log", jbossHome,
+                (jbossHome.endsWith(File.separator) || jbossHome.endsWith("/")) ? "" : File.separator,
+                File.separator, File.separator);
+        logPath = logPath.replace('/', File.separatorChar);
```

Comment:
It might be a good idea to lookup the path via a management operation instead. Something like:
```
final ModelNode address = Operations.createAddress("subsystem", "logging", "periodic-rotating-file-handler", "FILE");
final ModelNode op = Operations.createOperation("resolve-path", address);
final ModelNode result = client.execute(op);
if (!Operations.isSuccessfulOutcome(result)) {
    Assert.fail(Operations.getFailureDescription(result).asString());
}
final Path logPath = Paths.get(Operations.readResult(result).asString());
```

This would also allow the change to the `testsuite/integration/pom.xml` unnecessary.